pkg/rhcos/baremetal.go

```diff
@@ -0,0 +1,31 @@
+package rhcos
+
+import (
+	"context"
+	"net/url"
+
+	"github.com/pkg/errors"
+)
+
+// Baremetal fetches the URL of the Red Hat Enterprise Linux CoreOS release.
+func Baremetal(ctx context.Context) (string, error) {
```

Comment:
other functions in the package are based on type of os-image and not the platform. This does not fall into that.

What is the type of rhcos image that Baremetal platform will use.. and naming it based on that would be more desirable.

Reply:
Ack, renamed from baremetal.go/Baremetal to openstack.go/OpenStack, and moved the comment to the asset where we consume this.